Makefile

```diff
@@ -4,54 +4,29 @@
 # -------------------------------------------------------------
 # This makefile defines the following targets
 #
-#   - ca                           - clones the fabric-ca repository.
-#   - ci-smoke                     - update submodules, clone fabric, pulls docker images and executes smoke tests.
-#   - ci-barebones                 - update submodules, clone fabric, pulls docker images and executes barebones tests.
-#   - ci-daily                     - update submodules, clone fabric, pulls docker images and executes daily test suite.
-#   - k8s-sys-test                 - Triggers system tests on k8s cluster
-#   - build-docker-images          - builds fabric & ca docker images.
-#   - build-fabric                 - builds fabric docker images and binaries.
-#   - build-fabric-ca              - builds fabric-ca docker images and binaries.
-#   - fabric                       - clones fabric repository.
-#   - fabric-chaincode-java        - clones the fabric-chaincode-java repository.
-#   - smoke-tests                  - runs Smoke Test Suite.
-#   - barebones-tests              - runs Barebones Test Suite.
-#   - daily-tests                  - runs Daily Test Suite.
-#   - pull-images                  - pull the images and binaries from Artifactory.
-#   - javaenv                      - clone the fabric-chaincode-java repository and build the javaenv image.
-#   - nodeenv                      - clone the fabric-chaincode-node repository and build the nodeenv image.
-#   - svt-weekly-pte-12hr-test-k8s - Test 12hr longrun test in k8s environment.
-#   - npm-init                     - initialize the PTE NPM modules
-#   - git-latest                   - init git submodules to latest available commit.
-#   - git-init                     - init git submodules.
-#   - pre-setup                    - installs node and govendor
-#   - clean                        - cleans the docker containers.
-#   - gotools                      - installs go tools, such as: ginkgo, golint, goimports, gocov and govendor
+#   - regression/barebones			- Executes barebones tests
```

Comment:
Alignment is missing for descriptions.

Reply:
It's only in the github UI, its correct in the actual Makefile, I've been noticing this in a lot of places lately in GitHub